package.json

```diff
@@ -22,20 +22,23 @@
   ],
   "dependencies": {
     "apollo-client": "^1.4.2",
-    "babel-runtime": "^6.23.0"
+    "babel-preset-env": "^1.5.2",
```

Comment:
No, if move them back to dev, i need to install all of them to build on react-native.